addons/advanced_fatigue/CfgEden.hpp

```diff
@@ -0,0 +1,58 @@
+class Cfg3DEN {
+    class Attributes {
+        class Slider;
+        class GVAR(slider): Slider {
+            attributeLoad = " \
+                diag_log (""attributeLoad: "" + str _this); \
+                params [""_ctrlGroup""]; \
+                private _slider = _ctrlGroup controlsGroupCtrl 100; \
+                private _edit = _ctrlGroup controlsGroupCtrl 101; \
+                _slider sliderSetPosition _value; \
+                _edit ctrlSetText ([_value, 1, 1] call CBA_fnc_formatNumber); \
+            ";
+            attributeSave = " \
+                diag_log (""attributeSave: "" + str _this); \
+                params [""_ctrlGroup""]; \
+                sliderPosition (_ctrlGroup controlsGroupCtrl 100); \
+            ";
+            onLoad = " \
+                diag_log (""onLoad: "" + str _this); \
+                params [""_ctrlGroup""]; \
+                private _slider = _ctrlGroup controlsGroupCtrl 100; \
+                private _edit = _ctrlGroup controlsGroupCtrl 101; \
+                _slider sliderSetRange [0, 2]; \
+                _slider ctrlAddEventHandler [""SliderPosChanged"", { \
+                    params [""_slider""]; \
+                    private _edit = (ctrlParentControlsGroup _slider) controlsGroupCtrl 101; \
+                    private _value = sliderPosition _slider; \
+                    _edit ctrlSetText ([_value, 1, 1] call CBA_fnc_formatNumber); \
+                }]; \
+                _edit ctrlAddEventHandler [""KillFocus"", { \
+                    params [""_edit""]; \
+                    private _slider = (ctrlParentControlsGroup _edit) controlsGroupCtrl 100; \
+                    private _value = ((parseNumber ctrlText _edit) min 2) max 0; \
+                    _slider sliderSetPosition _value; \
+                    _edit ctrlSetText str _value; \
```

Comment:
I'd like to move the whole functionality of having Eden sliders with custom min and max values somewhere else eventually. Probably CBA. I think it's fine for now though - I don't like cluttering the AF functions with Eden sillyness tbh.
Thanks for spotting those `diag_log`s, will fix.